chain/network/src/routing/routing_table_actor.rs

```diff
@@ -408,10 +429,38 @@ impl RoutingTableActor {
 impl Handler<StopMsg> for RoutingTableActor {
     type Result = ();
     fn handle(&mut self, _: StopMsg, _ctx: &mut Self::Context) -> Self::Result {
+        self.edge_verifier_pool.do_send(StopMsg {});
         System::current().stop();
     }
 }
 
+impl Handler<EdgeList> for RoutingTableActor {
+    type Result = bool;
+
+    #[perf]
+    fn handle(&mut self, msg: EdgeList, ctx: &mut Self::Context) -> Self::Result {
+        self.edge_verifier_requests_in_progress += 1;
+        let mut msg = msg;
+        msg.edges.retain(|x| self.is_edge_newer(x.key(), x.nonce()));
+        let peer_id = msg.peer_id.clone();
+        self.edge_verifier_pool
+            .send(msg)
+            .into_actor(self)
+            .map(move |res, act, _| {
+                act.edge_verifier_requests_in_progress -= 1;
+
+                if let Ok(false) = res {
+                    act.peers_to_ban.push(peer_id);
+                }
+                // TODO missing ban peer logic
```

Comment:
what is the 'ban peer logic' that is missing - can you explain more?

Reply:
See updated comments in https://github.com/near/nearcore/pull/5429

Reply:
Sorry, that's an old comment which was there while I was rewriting the code.
Right now `RoutingTableActor` sends to `PeerManagerActor` periodically, list of peers to ban, for sending us invalid edges.